lib/routes/brain.rb

```diff
@@ -27,30 +27,29 @@ def self.registered(slapi)
           # Saves into brain as Plugin: plugin name (hash), key, value
           slapi.save(params[:plugin], params[:key], params[:value])
           status 200
-          { 'message' => 'it worked' }.to_json
         end
 
         # Handles a POST request for '/v1/delete'
         #
         # @param [Hash] params the parameters sent on the request
         # @option params [String] :plugin Utilizes the plugin name as Hash Name
         # @option params [String] :key Key you wish to delete (will delete value as well)
-        # @return [String] status or message "it worked"
+        # @return [Integer] status
         slapi.post '/v1/delete' do
           raise 'missing plugin name' unless params[:plugin]
           raise 'missing key' unless params[:key]
 
           # Saves into brain as Plugin: plugin name (hash), key
           slapi.delete(params[:plugin], params[:key])
           status 200
```

Comment:
If there is no longer any content, then maybe a 204 is appropriate?  

```
9.7 DELETE

A successful response SHOULD be 200 (OK) if the response includes an entity describing the status, 202 (Accepted) if the action has not yet been enacted, or 204 (No Content) if the action has been enacted but the response does not include an entity. 
```

However this is not a DELETE, but a POST, so maybe 201 is appropriate, but "Created" seems weird here.